web/static/js/thesis-editor.js

```diff
@@ -241,14 +331,16 @@ class ThesisEditor extends React.Component {
   }
 
   render () {
+    if (this.state.deleted) { return <div id="thesis"></div> }
+
     return (
       <div id="thesis">
         <div id='thesis-editor' className={this.renderEditorClass()}>
+          {this.dynamicEnabled() ? <AddButton onPress={this.addPagePressed} /> : null}
+          {this.state.template ? <DeleteButton onPress={this.deletePagePressed} /> : null}
```

Comment:
Only show Add if we can actually add a new page. Only show Delete if we're on a dynamic page.